sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1097,3 +1097,28 @@ def test_vectorizers_invalid_ngram_range(vec):
     if isinstance(vec, HashingVectorizer):
         assert_raise_message(
             ValueError, message, vec.transform, ["good news everyone"])
+
+
+@pytest.mark.parametrize("vec", [CountVectorizer()])
+def test_countvectorizer_sort_features_64bit_sparse_indices(vec):
+    # If a count vectorizer has to store >= 2**31 count values, the sparse
+    # storage matrix has 64bit indices / indptrs.  This requires ~2*8*2**31
+    # bytes of memory in practice, so we just test the method that would
+    # hypothetically fail.
+
+    X = sparse.csr_matrix((5, 5), dtype=np.int64)
+
+    # force indices and indptr to int64.
+    INDEX_DTYPE = np.int64
```

Comment:
`INDICES_DTYPE` if we want to be consistent

Reply:
Sure, will update.